Расширил класс RoomHelper

KunakbaevAV opened the pull request:
1. Добавил инициацию первоначального загрузчика базы данных в App
2. В первоначальном загрузчике добавил заполнение нужных списков в БД
3. Расширил класс RoomHelper новыми методами, получающими нужные списки из БД

KunakbaevAV committed:
1. Добавил иннициацию первоначального загрузчика базы данных в App
2. В первоначальном загрузчике добавил заполнение нужных списков в БД
3. Расширил класс RoomHelper новыми методами, получающими нужные списки из БД

AlinaRakhimova commented on app/src/main/java/ru/geekbrains/psy_journal/di/App.java:
```diff
@@ -16,5 +18,12 @@ public void onCreate() {
+    }
+
+    private void firstInit() {
+        DataBaseFirstLoader dataBaseFirstLoader = new DataBaseFirstLoader();
```
Надо, наверное, Dagger-ом заинжектить

AlinaRakhimova commented on app/src/main/java/ru/geekbrains/psy_journal/model/database/DataBaseFirstLoader.java:
```diff
@@ -119,5 +134,27 @@ public void initDataBase() {
         tdDao.insert(new TD(74, "B/05.7.6", "Осуществление с целью профориентации комплекса диагностических мероприятий по изучению мотивации, личностных, характерологических особенностей лиц с ограниченными возможностями здоровья, обучающихся, испытывающих трудности в освоении основных общеобразовательных программ, развитии и социальной адаптации, в том числе несовершеннолетних обучающихся, приз ...
         tdDao.insert(new TD(75, "B/05.7.7", "Ведение профессиональной документации (планы работы, протоколы, журналы, психологические заключения и отчеты)", 12));
+
+        categoryDao.insert(new Category(1, "Классные руководители"));
```
Здесь по идее надо использовать RoomHelper уже. Получается мы создаем объекты интерфейсов, а не классов

KunakbaevAV replied:
Переделал с использованием RoomHelper

KunakbaevAV committed:
Улучшил код после ревью
1. Расширил RoomHelper
2. Использовал расширенный RoomHelper для первоначального заполнения базы данных классом DataBaseFirstLoader
3. Заинжектил DataBaseFirstLoader в App
